ospool.osg-htc.org/fe-admin

```diff
@@ -42,7 +42,11 @@ def load_data():
     '''
     Load data from topology database
     '''
-    url = "https://topology.opensciencegrid.org/rgsummary/xml?service=on&service_109=on"
+    url = "https://my.opensciencegrid.org/rgsummary/xml?" + \
+          "gridtype=on&gridtype_1=on&" + \
+          "disable=on&disable_value=0&" + \
+          "active=on&active_value=1&" + \
+          "service=on&service_109=on"
```

Comment:
```suggestion
    url = "https://my.opensciencegrid.org/rgsummary/xml?" \
          "gridtype=on&gridtype_1=on&" \
          "disable=on&disable_value=0&" \
          "active=on&active_value=1&" \
          "service=on&service_109=on"
```
strings next to each other get concatenated so you don't need the `+`

Reply:
Ah right, thanks